core/src/main/java/io/questdb/griffin/SqlKeywords.java

```diff
@@ -360,10 +360,10 @@ public static boolean isHourKeyword(CharSequence tok) {
         }
 
         int i = 0;
-        return (tok.charAt(i++) | 32) == 'h'
-                && (tok.charAt(i++) | 32) == 'o'
-                && (tok.charAt(i++) | 32) == 'u'
-                && (tok.charAt(i) | 32) == 'r';
+        return (tok.charAt(i++)) == 'h'
```

Comment:
Function names are case sensitive. Reason i changed that is because this SQL would work:
```sql
select HOUR(ts), count() from tab
```

and this SQL would not:
```sql
select HOUR(ts), count() from tab where y > 0.5
```

Reply:
great